src/Model/Model.ts

```diff
@@ -70,13 +70,13 @@ export class Model<T extends IModelData> implements IModel<T> {
   public readonly validationRules: IModelKeys<T>;
   public readonly relationships: { [key: string]: IFieldRelationship };
 
-  @attr(StringField, { serialize: false })
+  @attr(StringField, { serialize: false, readOnly: true })
```

Comment:
These changes weren't really part of this ticket but they should have been, it's using that readOnly property.